public/app/controllers/userctrl.client.js

```diff
@@ -12,8 +23,8 @@ angular.module('hairvenApp')
       $scope.signin = function() {
 
         var data = {
-          username: $scope.username,
-          password: $scope.password
+          username: $scope.user_username,
+          password: $scope.user_password
```

Comment:
Identifier 'user_password' is not in camel case.

Reply:
Remember this. camel case?? @akinjide